ontology/yaml/resources/HVAC/entity_types/MAU.yaml

```diff
@@ -632,6 +632,20 @@ MAU_SFSS2X_SFVSC2X_CHWSC_HWSC_FDPM_SSPC_STC_VOADM2X:
   - SSPC
   - STC
   - VOADM2X
+  
+MAU_SFSS_SFVSC_SPSS_STM_SHM_SFC_FDPSM:
```

Comment:
please double-check this type name, it does not match the listed types in the implements section

Reply:
Done